scripts/download_backtest_data.py

```diff
@@ -27,15 +30,36 @@
 
 print(f'About to download pairs: {PAIRS} to {dl_path}')
 
-# Init Bittrex exchange
+# Init exchange
 exchange._API = exchange.init_ccxt({'key': '',
                                     'secret': '',
-                                    'name': 'bittrex'})
+                                    'name': args.exchange})
 
 for pair in PAIRS:
     for tick_interval in TICKER_INTERVALS:
         print(f'downloading pair {pair}, interval {tick_interval}')
-        data = exchange.get_ticker_history(pair, tick_interval)
+
+        since_time = None
+        if args.days:
+            since_time = int((time.time() - args.days * DAY_SECCONDS) * 1000)
```

Comment:
could be simplified to `arrow.utcnow().shift(days=-args.days).timestamp * 1000` - removing the need for `MIN_SECCONDS` and `DAY_SECCONDS` and `HOUR_SECCONDS`